paywall/src/utils/postOffice.ts

```diff
@@ -129,6 +138,82 @@ export function setupPostOffice<T extends MessageTypes = MessageTypes>(
   }
 }
 
+export function postMessageIsSafe(
```

Comment:
Extracted from the head of `mainWindowPostOffice` -- makes sense to have this as a separate function, and now when debug mode is on you'll be able to see why a message gets rejected.